types/bloem/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for bloem 0.2
+// Project: https://github.com/wiedi/node-bloem
+// Definitions by: Daniel Byrne <https://github.com/danwbyrne>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types= "node" />
+
+export as namespace bloem;
```

Comment:
I believe this namespace is not necessary. Check [Needles Namespacing](http://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html) on the TypeScript Handbook for details.